### ISSUE TITLE
When I run the train.py,the results are "loss nan - iou 0.0000 - val_loss nan - val_iou 0.0000"?

### ISSUE THREAD

**Issue by wxyWendy** [4uiiurz1/pytorch-nested-unet#49]
When I run the train.py,why the results are "loss nan - iou 0.0000 - val_loss nan - val_iou 0.0000"?I run on the dataset dsb2018_96. 
loss nan - iou 0.0000 - val_loss nan - val_iou 0.0000
Epoch [93/100]
100%|███████████████████████████████████████████████████████████| 33/33 [00:01<00:00, 18.80it/s, loss=nan, iou=5.04e-10]
100%|█████████████████████████████████████████████████████████████| 9/9 [00:00<00:00, 36.14it/s, loss=nan, iou=5.76e-10]
loss nan - iou 0.0000 - val_loss nan - val_iou 0.0000
Epoch [94/100]
100%|███████████████████████████████████████████████████████████| 33/33 [00:01<00:00, 18.72it/s, loss=nan, iou=5.01e-10]
100%|█████████████████████████████████████████████████████████████| 9/9 [00:00<00:00, 36.95it/s, loss=nan, iou=5.76e-10]
loss nan - iou 0.0000 - val_loss nan - val_iou 0.0000
Epoch [95/100]
100%|███████████████████████████████████████████████████████████| 33/33 [00:01<00:00, 18.77it/s, loss=nan, iou=5.14e-10]
100%|█████████████████████████████████████████████████████████████| 9/9 [00:00<00:00, 36.63it/s, loss=nan, iou=5.76e-10]
loss nan - iou 0.0000 - val_loss nan - val_iou 0.0000
Epoch [96/100]
100%|███████████████████████████████████████████████████████████| 33/33 [00:01<00:00, 18.76it/s, loss=nan, iou=5.06e-10]
100%|█████████████████████████████████████████████████████████████| 9/9 [00:00<00:00, 36.84it/s, loss=nan, iou=5.76e-10]
loss nan - iou 0.0000 - val_loss nan - val_iou 0.0000
Epoch [97/100]
100%|███████████████████████████████████████████████████████████| 33/33 [00:01<00:00, 18.74it/s, loss=nan, iou=5.11e-10]
100%|█████████████████████████████████████████████████████████████| 9/9 [00:00<00:00, 37.12it/s, loss=nan, iou=5.76e-10]
loss nan - iou 0.0000 - val_loss nan - val_iou 0.0000
Epoch [98/100]
100%|███████████████████████████████████████████████████████████| 33/33 [00:01<00:00, 18.79it/s, loss=nan, iou=5.05e-10]
100%|█████████████████████████████████████████████████████████████| 9/9 [00:00<00:00, 36.90it/s, loss=nan, iou=5.76e-10]
loss nan - iou 0.0000 - val_loss nan - val_iou 0.0000
Epoch [99/100]
100%|████████████████████████████████████████████████████████████| 33/33 [00:01<00:00, 18.77it/s, loss=nan, iou=5.2e-10]
100%|█████████████████████████████████████████████████████████████| 9/9 [00:00<00:00, 37.04it/s, loss=nan, iou=5.76e-10]
loss nan - iou 0.0000 - val_loss nan - val_iou 0.0000

**Comment by wxyWendy** [comment]
And I use deep supervision.It means that I set deep_supervision as True(Not False) on train.py.

**Comment by yjsb** [comment]
Me too, can u get me ur answer?
